src/components/Button.tsx

```diff
@@ -130,6 +130,7 @@ export interface ButtonConfig extends Base<HTMLButtonElement> {
    * Borderless is exclusively a ripple element
    */
   type?: "standard" | "flat" | "borderless";
+  className?: string;
```

Comment:
`Base<HTMLButtonElement>` already has className defined there so no need for this definition